src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/HealthCheckSpecification.groovy

```diff
@@ -64,5 +69,27 @@ class HealthCheckSpecification extends BaseSpecification {
             }
         }
         regionVerifications.verify()
+
+        and: "Feature toggles are in expected state"
+        verifyAll(northbound.getFeatureToggles()) {
+            flowsRerouteOnIslDiscoveryEnabled
+            createFlowEnabled
+            updateFlowEnabled
+            deleteFlowEnabled
+            useBfdForIslIntegrityCheck
+            floodlightRoutePeriodicSync
+            server42FlowRtt
+            //for below props any value allowed. dependent tests will skip themselves or adjust if feature is off
+            //flowsRerouteUsingDefaultEncapType
+            //collectGrpcStats
+        }
+
+        and: "Switches configurations are in expected state"
+        topology.switches.each { sw ->
+            verifyAll(northbound.getSwitchProperties(sw.dpId)) {
+                multiTable == useMultitable
```

Comment:
It won't work for hardware env.
Some switch doesn't support the `multiTable` mode
```
topology.activeSwitches.each {sw ->
            if (sw.features.contains(SwitchFeature.MULTI_TABLE)) {
                verifyAll(northbound.getSwitchProperties(sw.dpId)) {
                    multiTable == useMultitable
                }
            }
        }
```

Reply:
changed to
```
            verifyAll(northbound.getSwitchProperties(sw.dpId)) {
                multiTable == useMultitable && sw.features.contains(SwitchFeature.MULTI_TABLE)
```